utils/cr_demo/run_demo.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+dmg_pool_list=`dmg pool list`
+dmg_check_enable=`dmg check enable`
+dmg_check_start=`dmg check start`
```

Comment:
(lint) dmg_check_start appears unused. Verify use (or export if used externally). [SC2034]
(lint) Use $(...) notation instead of legacy backticked `...`. [SC2006]